src/common/byteArrayHelper/tests/CMakeLists.txt

```diff
@@ -38,7 +38,7 @@ if(CMAKE_SYSTEM_NAME STREQUAL "Windows")
         pthread
         crypto
         ssl
-        -static-libgcc -static-libstdc++
+        -static-libgcc
```

Comment:
For these and all the other occruances of removing libstdc++ as a static library. Is this ok? afaik there are other efforts to compile the agent with these libraries linked statically.

Reply:
It does not exist in MSVC and creates a warning every time it is used indicating that it will be ignored.